### PR TITLE
MEN-1006: Increased the standard mender-retry-polling interval

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog:Title

Signed-off-by: Ole Petter <ole.orhagen@cfengine.com>

### CODE REVIEW

File: meta-mender-demo/recipes-mender/mender/mender_%.bbappend [hunk 1]
```diff
@@ -4,7 +4,7 @@ SRC_URI_append = " file://server.crt"
 
 MENDER_UPDATE_POLL_INTERVAL_SECONDS = "5"
 MENDER_INVENTORY_POLL_INTERVAL_SECONDS = "5"
-MENDER_RETRY_POLL_INTERVAL_SECONDS = "1"
+MENDER_RETRY_POLL_INTERVAL_SECONDS = "10"
```
Comment (pasinskim) [inline meta-mender-demo/recipes-mender/mender/mender_%.bbappend]: Nice and simple, but I think we've been considering increasing this to 30 seconds.

**Comment by oleorhagen** [comment]
@pasinskim fixed

**Comment by oleorhagen** [comment]
Done

**Comment by kacf** [comment]
Rerunning tests... again!

**Comment by kacf** [comment]
Hmm, there is still some failures here. Everything with "pull/447/head" in them can be ignored (they are a remnant of earlier checking), but even master fails.

Will rerun yet again and compare with [this one](https://s3-eu-west-1.amazonaws.com/mender-testing-reports/integration-reports/1205/report.html).

**Comment by kacf** [comment]
Dammit, `test_bootstrapping` is failing *consistently* and I don't understand why.

From the output it looks like it is using the pure Fabric `run` function, but it should be using the `run` function from our `common` module, which may explain why it times out after a very short delay. It is capped at 30 minutes, whereas this test times out after less than 3 minutes, which may explain the issue; maybe it simply haven't had time to establish the connection. I will try to investigate why, but I may not have time before the summit.

**Comment by oleorhagen** [comment]
If u'd like, I can have a look on friday.

**Comment by kacf** [comment]
Feel free! Notice in particular:

```
/usr/local/lib/python2.7/dist-packages/fabric/tasks.py:173: in run
return self.wrapped(*args, **kwargs)
tests/test_bootstrapping.py:53: in accept_devices
run("strings /data/mender/mender-store | grep -q 'authtoken'")
```

.. how it goes straight from `accept_devices` function into the `run` function in Fabric. There should be an intermediary function in `common.py` there, which I thought would be ensured by the import order, but I guess not.

**Comment by GregorioDiStefano** [comment]
Should be fixed now.

**Pull request merged**